crates/lib/src/filterable_value.rs

```diff
@@ -87,7 +110,7 @@ pub enum TermBound<T> {
 }
 impl<Bound: FilterableValue> TermBound<&Bound> {
     #[inline]
-    /// If `self` is [`TermBound::Range`], returns the `rend_idx` value for [`BTreeScanArgs`],
+    /// If `self` is [`TermBound::Range`], returns the `rend_idx` value for `IndexScanRangeArgs`,
```

Comment:
This change to quiet a warning from `cargo doc`. `BTreeScanArgs` is no longer called that, and also is not available in this crate.